packages/manager/apps/pci-ai-tools/src/components/order/preset-image/PresetImageSelect.component.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  Card,
+  CardContent,
+  CardHeader,
+  RadioGroup,
+  RadioTile,
+  Separator,
+} from '@datatr-ux/uxlib';
+import {
+  ArrowUpRightFromSquare,
+  ArrowUpRightFromSquareIcon,
+  Wrench,
+} from 'lucide-react';
+import A from '@/components/links/A.component';
+import ai from '@/types/AI';
+
+interface PresetImageSelectProps {
+  images: ai.job.PresetImage[];
+  value: string;
+  onChange: (newImage: string) => void;
+}
+
+const PresetImageSelect = React.forwardRef<
+  HTMLInputElement,
+  PresetImageSelectProps
+>(({ images, value, onChange }, ref) => {
+  const { t } = useTranslation('ai-tools/components/preset-image');
+  return (
+    <Card data-testid="preset-image-select">
+      <CardHeader>
+        <h5>{t('presetImageTitle')}</h5>
+        <div className="pt-4 text-sm">
+          <p>{t('presetImageDesc')}</p>
+          <p>
+            {t('missingImageDesc')}
+            <A
+              className="ml-2"
+              href={'https://discord.com/invite/vXVurFfwe9'}
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              {t('discordLinkLabel')}
+              <ArrowUpRightFromSquareIcon className="size-4 inline ml-1 mb-1" />
+            </A>
+          </p>
+        </div>
+      </CardHeader>
+      <CardContent className="space-y-2">
+        <RadioGroup
+          data-testid="images-select-container"
+          ref={ref}
+          className="grid grid-cols-1 md:grid-cols-2 xl:grid-cols-3 gap-2"
+          value={value}
+          onValueChange={onChange}
+        >
+          {images.map((image) => (
+            <RadioTile
+              data-testid={`image-radio-tile-${image.id}`}
+              key={image.id}
+              onChange={() => {
+                onChange(image.id);
+              }}
+              value={image.id}
+              checked={image.id === value}
+            >
```

Comment:
`onChange` and `checked` are handled internally by the component

```suggestion
            <RadioTile
              data-testid={`image-radio-tile-${image.id}`}
              key={image.id}
              value={image.id}
            >
```